views/layout.pug

```diff
@@ -67,6 +67,14 @@ html
           span.navbar-toggler-icon
         #navbarSupportedContent.collapse.navbar-collapse.justify-content-between
           ul.navbar-nav#navbar_content
+          #select_experiment
+            .dropdown
+              a.btn.btn-warning.dropdown-toggle(role='button' data-bs-toggle='dropdown' onchange="ChangeExperiment()") #{experiment}
```

Comment:
Why btn-warning yellow, not the nice blue we have elswehere? It makes me nervous seeing the bright colours XD